src/definitions/chat.json

```diff
@@ -2,95 +2,116 @@
+		"fr": "J'ai modifié le message.",
+		"pt-BR": "Editei a mensagem."
 	},
 	"bareReply": {
 		"en-US": "I have sent the message.",
-		"fr": "J'ai envoyé le message."
+		"fr": "J'ai envoyé le message.",
+		"pt-BR": "Enviei a mensagem."
 	},
 	"noPrivacyReply": {
 		"en-US": "I can not reply to you in this channel.\nPlease ask me in a private channel instead.",
-		"fr": "Je ne peux pas te répondre dans ce salon.\nMerci de me demander dans un salon privé à la place."
+		"fr": "Je ne peux pas te répondre dans ce salon.\nMerci de me demander dans un salon privé à la place.",
+		"pt-BR": "Não posso te responder nesse canal.\nPor favor, me pergunte em um canal privado em vez disso."
 	},
 	"noChannelReply": {
 		"en-US": "I do not know any channel with this tag.",
-		"fr": "Je ne connais aucun salon avec cette étiquette."
+		"fr": "Je ne connais aucun salon avec cette étiquette.",
+		"pt-BR": "Não conheço nenhum canal com essa marcação."
 	},
 	"noMessageReply": {
 		"en-US": "I do not know any message with this identifier in this channel.",
-		"fr": "Je ne connais aucun message avec cet identifiant dans ce salon."
+		"fr": "Je ne connais aucun message avec cet identifiant dans ce salon.",
+		"pt-BR": "Não sei nenhuma mensagem com essa identificação nesse canal."
 	},
 	"noPositionReply": {
 		"en-US": "I do not know any slot with this position.\nPlease give me a position between `0` and `$<max>` instead.",
-		"fr": "Je ne connais aucun emplacement avec cette position.\nMerci de me donner une position entre `0` et `$<max>` à la place."
+		"fr": "Je ne connais aucun emplacement avec cette position.\nMerci de me donner une position entre `0` et `$<max>` à la place.",
+		"pt-BR": "Não sei nenhum espaço com essa posição.\nPor favor, me dê uma posição entre `0` e `$<max>` em vez disso."
 	},
 	"noInteractionReply": {
 		"en-US": "I can not edit interaction replies or follow-ups.",
-		"fr": "Je ne peux pas modifier de réponses ou de suites aux interactions."
+		"fr": "Je ne peux pas modifier de réponses ou de suites aux interactions.",
+		"pt-BR": "Eu não posso editar respostas de interação ou acompanhamentos."
 	},
 	"noContentOrAttachmentReply": {
 		"en-US": "Please keep a content or attachments.",
-		"fr": "Merci de garder un contenu ou des pièces jointes."
+		"fr": "Merci de garder un contenu ou des pièces jointes.",
+		"pt-BR": "Por favor, mantenha um conteúdo ou anexos."
 	},
 	"noPatchPermissionReply": {
 		"en-US": "I do not have the rights to edit this message.",
-		"fr": "Je n'ai pas les droits pour modifier ce message."
+		"fr": "Je n'ai pas les droits pour modifier ce message.",
+		"pt-BR": "Eu não tenho as permissões pra editar essa mensagem."
 	},
 	"noPostPermissionReply": {
 		"en-US": "I do not have the rights to send this message.",
-		"fr": "Je n'ai pas les droits pour envoyer ce message."
+		"fr": "Je n'ai pas les droits pour envoyer ce message.",
+		"pt-BR": "Eu não tenho as permissões pra editar essa mensagem."
```

Comment:
The translation is identical to the one above. Should it be different?

Reply:
oh ye;; oops